### ISSUE TITLE
Little rendenring problem with Image32

### ISSUE THREAD

**Issue by carloBarazzetta** [EtheaDev/SVGIconImageList#219]
Two icons with some little problems using Image32:
![image](https://user-images.githubusercontent.com/7803548/155950744-3b1977c0-3a0b-4ff7-b677-bb1e7ce1b348.png)
The source files:
![apps_adblock_plus](https://user-images.githubusercontent.com/7803548/155950954-c6d236ec-0050-46c4-b6fd-76f22d025892.svg)
![apps_antennapod](https://user-images.githubusercontent.com/7803548/155950958-d43fdd0f-e795-4b98-89af-fb7b2b8c7ed0.svg)

**Comment by AngusJohnson** [comment]
Yep. A little while ago I "tidied up" some code but evidently broke the line ending stuff 😱. 
I will "untidy" things 😜.

**Comment by AngusJohnson** [comment]
Should be fixed now:
https://sourceforge.net/p/image32/code/ci/master/tree/source/

(Rounded line endings were not being drawn whenever no-round edge joins had been selected.)
